Jenkinsfile

```diff
@@ -0,0 +1,3 @@
+// Builds a module using https://github.com/jenkins-infra/pipeline-library
+// TODO: Explicitly require LTS baseline once SSHD Module 2.x gets into LTS
```

Comment:
What does this mean? By default we build against whatever baseline is listed in this module’s POM. You only need to patch `Jenkinsfile` if you wish to build & test against multiple/alternate baselines.